packages/aws-cdk-lib/aws-cloudwatch/test/graphs.test.ts

```diff
@@ -829,6 +829,34 @@ describe('Graphs', () => {
     }]);
   });
 
+  test('add displayLabelsOnChart to GraphWidget', () => {
```

Comment:
Let's add a test which verifies what happens when the graph type is not a pie chart.